frontend/packages/core/src/tests/tab.test.tsx

```diff
@@ -1,43 +1,28 @@
 import React from "react";
-import { mount, shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
 
-import { Tab, Tabs } from "../tab";
+import "@testing-library/jest-dom";
 
-describe("Tabs component", () => {
-  describe("with a value set", () => {
-    let component;
-    beforeEach(() => {
-      component = shallow(
-        <Tabs value={1}>
-          <Tab label="meow" />
-          <Tab label="mix" />
-        </Tabs>
-      );
-    });
+import { Tab, Tabs } from "../tab";
 
-    it("renders", () => {
-      expect(component.find(Tabs)).toBeDefined();
-    });
+beforeEach(() => {
+  render(
+    <Tabs value={1}>
+      <Tab label="meow" />
+      <Tab label="mix" />
+    </Tabs>
+  );
+});
 
-    it("renders children tabs", () => {
-      expect(component.find(Tab)).toHaveLength(2);
-    });
+test("renders correctly", () => {
+  expect(screen.getByTestId("styled-tabs")).toBeInTheDocument();
```

Comment:
We should probably make this `.toBeVisible()` so that we can make sure it's not only in the DOM but also visible to the user.